File: June-18/17-06-18/c++/01-Maimum-sum-of-building-sums.cpp
#include<bits/stdc++.h>

using namespace std;

int main()
{
    int n;
    cin >> n;
    int ar[2*n];
    for(int i = 0; i < 2*n; i++)
        cin >> ar[i];
    sort(ar,ar+(2*n));
    int ans = 0;
    for(int i = 0; i < 2*n; i += 2)
        ans += ar[i];
    cout << ans << endl;
}

File: June-18/17-06-18/java/01-Maimum-sum-of-building-sums.java

import java.util.*;

class TestClass {
    public static void main(String args[] ) throws Exception {
        Scanner input = new Scanner(System.in);
        int n = input.nextInt();
        n = n * 2;
        int ar[] = new int[n];
        for(int i = 0; i < n; i++)
            ar[i] = input.nextInt();
        Arrays.sort(ar);
        int ans = 0;
        for(int i = 0; i < n; i+=2)
            ans += ar[i];
        System.out.println(ans);
        
    }
}
